web_widget_dropdown_dynamic/static/src/js/basic_model.js

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2019-2020 Brainbean Apps (https://brainbeanapps.com)
+ * License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+ */
+odoo.define("web_widget_dropdown_dynamic.basic_model", function (require) {
```

Comment:
any reason to keep this as old style JS?

Reply:
I just updated it, please check again